instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/RumTestUtils.kt

```diff
@@ -123,6 +123,12 @@ private fun JsonObject.verifyEventMatches(event: ExpectedViewEvent) {
     if (viewArguments.isNotEmpty()) {
         assertThat(this.getAsJsonObject(CONTEXT_KEY)).containsAttributes(viewArguments)
     }
+    if (event.sessionIsActive != null) {
```

Comment:
shouldn't they always have this field?

Reply:
I guess for ViewEvents yes they should always have this field. I'll fix.